regression/storage.js

```diff
@@ -370,7 +370,7 @@ describe('storage', function() {
 
         file.download(function(err, remoteContents) {
           assert.ifError(err);
-          assert.equal(fileContents, remoteContents);
+          assert.equal(String(fileContents), String(remoteContents));
```

Comment:
This popped up for me. `fileContents` and `remoteContents` were buffers, which are problematic to compare. We do have a utility library we could use, but it's just as easy to convert to Strings to compare.